tools/testbench/makefile-xcc

```diff
@@ -0,0 +1,127 @@
+# Build xt-run version of testbench
+
+CC = xt-xcc
+LD = xt-xc++
+OBJDUMP = xt-objdump
+OPTFLAGS = -std=gnu99 -fgnu89-inline -g -O2 -Wall -Wmissing-prototypes \
+	 -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast \
+	 -Wpointer-arith -Werror
+INSTALLPATH = build_testbench/install/bin
+INCLUDEPATHS = -I./include -I../tplg_parser/include \
+	     -I../../src/arch/host/include \
+	     -I../../src/platform/library/include \
+	     -I../../src/include \
+	     -I../../xtos/include
+DEFINES = -DCONFIG_IPC_MAJOR_3 -DCONFIG_CORE_COUNT=1 \
+	-DCONFIG_SYSTICK_PERIOD=1 -DCONFIG_LIBRARY \
+	-DCONFIG_LIBRARY_STATIC -DCONFIG_SOF_STACK_SIZE=4096 \
+	-DRELATIVE_FILE -DCONFIG_FORMAT_S16LE -DCONFIG_FORMAT_S24_4LE \
+	-DCONFIG_FORMAT_S32LE -DCONFIG_COMP_SRC_IPC4_FULL_MATRIX \
```

Comment:
Seems SRC set IPC4, not sure whether there is conflicts with CONFIG_IPC_MAJOR_3 CONFIG_COMP_SRC_IPC4_FULL_MATRIX

Reply:
The SRC profile name is confusing, maybe it could be changed. I came up with this name for a conversions matrix that matches requirements for SRC in IPC4 usage. This can be used despite the name for IPC3 and 4 and it doesn't change the SRC API. There's other CONFIG option for that.

Reply:
ok